packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -710,13 +713,13 @@ export async function launch({
    *
    * @param {InboundQueue} inboundQueue
    * @param {Cranker} runSwingset
-   * @param {CrankerPhase} phase
+   * @param {InboundQueueName} phase
    */
   async function processActions(inboundQueue, runSwingset, phase) {
     let keepGoing = true;
     for await (const { action, context } of inboundQueue.consumeAll()) {
       const inboundNum = `${context.blockHeight}-${context.txHash}-${context.msgIdx}`;
-      inboundQueueMetrics.decStat();
+      inboundQueueMetrics.decStat(phase);
```

Comment:
Should we pay attention to type as well?
```suggestion
      inboundQueueMetrics.decStat({ queue: phase, type: action.type });
```

Reply:
I don't think so because the type is not available when incrementing or setting the initial value.

Reply:
Excellent point.